custom_functions/filter_out_domain_list_from_url.json

```diff
@@ -0,0 +1,36 @@
+{
+    "create_time": "2020-07-07T19:26:39.571366+00:00",
+    "custom_function_id": "8d4c6549a9be019ed5fe5a6e42e462ad277284f4",
+    "description": "Input a list of urls and the name of a custom_list that contains safe domains. Output urls where the domain is NOT present in the custom_list. Useful for ensuring that you detonate only interesting urls.",
+    "draft_mode": false,
+    "inputs": [
+        {
+            "contains_type": [
+                "url"
+            ],
+            "description": "",
```

Comment:
Are we accepting http and https URLs only? What about FTP url's and other protocols? Maybe the description should be:

"One or more HTTP or HTTPS URLs to filter against the domain allowlist"